modules/contact-form/admin.php

```diff
@@ -602,24 +604,29 @@ function grunion_ajax_shortcode_to_json() {
 function grunion_ajax_spam() {
 	global $wpdb;
 
-	if ( empty( $_POST['make_it'] ) )
+	if ( empty( $_POST['make_it'] ) ) {
 		return;
+	}
 
 	$post_id = (int) $_POST['post_id'];
 	check_ajax_referer( 'grunion-post-status-' . $post_id );
-	if ( !current_user_can("edit_page", $post_id) )
+	if ( ! current_user_can("edit_page", $post_id) ) {
```

Comment:
Since we're updating styles, can you please add spaces for the function call as well?